<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ieeetodec</class>
 <widget class="QWidget" name="ieeetodec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="extractMantissaLabel">
     <property name="text">
      <string>Extract the mantissa</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="hexLabel">
     <property name="text">
      <string>IEEE-754 Hex</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="binaryDisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="binaryLabel">
     <property name="text">
      <string>Convert to Binary</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="biasLabel">
     <property name="text">
      <string>Determine Bias</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="deNormMantissaLabel">
     <property name="text">
      <string>De-normalise</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="difficultyLabel">
     <property name="text">
      <string>Difficulty</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="generatePushButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextPushButton">
      <property name="text">
       <string>Next Step</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resultPushButton">
      <property name="text">
       <string>Result</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="extractMantissaDisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="decimalLabel">
     <property name="text">
      <string>Convert to Decimal</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="biasDisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="decimalDisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="deNormMantissaDisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="hexDisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="difficultyComboBox">
     <item>
      <property name="text">
       <string>Easy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Moderate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hard</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="signLabel">
     <property name="text">
      <string>Check the sign bit</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="signDisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
